src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movies } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<Movies> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => <MovieCard movie={movie} />)}
```

Comment:
not fixed. key must be added on top level of mapping
```suggestion
    {movies.map(movie => <MovieCard movie={movie} key={movie.imdbId} />)}
```